anyway/infographics_utils.py

```diff
@@ -252,6 +252,89 @@ def get_accident_count_by_severity(location_info, start_time, end_time):
 _("Fatal, severe and light injured count in the specified years, split by injury severity")
 _("Fatal, severe and light accidents count in the specified years, split by accident severity")
 
+@register
+class SevereFatalCountByAccidentYearAndVehicleWidget(UrbanWidget):
+    name: str = "severe_fatal_count_by_vehicle_by_accident_year"
+
+    def __init__(self, request_params: RequestParams):
+        super().__init__(request_params, type(self).name)
+        self.rank = 15
+        self.text = {
+            "title": "Severe or fatal accidents on bikes, e-bikes, or scooters in "
```

Comment:
Title, and some other strings needs to be localized. You can look at other widgets code for how to localize.